Nightly Dotty workflow of 2025-05-03 failed

See https://github.com/scala/scala3/actions/runs/14806939134

publish failed
```
Version used by compiler to publish nightly release does not match expected pattern
```
because version string
```
Run ver=$(./project/scripts/sbt "print scala3-compiler-bootstrapped/version" | tail -n1)
  ver=$(./project/scripts/sbt "print scala3-compiler-bootstrapped/version" | tail -n1)
  echo "This build version: $ver"
  echo "THISBUILD_VERSION=$ver" >> $GITHUB_ENV
  shell: sh -e {0}
  env:
    DOTTY_CI_RUN: true
    DEVELOCITY_ACCESS_KEY: ***
    NIGHTLYBUILD: yes
    PGP_PW: ***
    PGP_SECRET: ***
    SONATYPE_PW: ***
    SONATYPE_USER: ***
java.lang.AssertionError: assertion failed: Invalid TASTy minor version
	at scala.Predef$.assert(Predef.scala:223)
	at Build$.checkReleasedTastyVersion(Build.scala:2554)
	at Build$.<init>(Build.scala:145)
	at Build$.<clinit>(Build.scala)
	at $92ac103c05862892d2d9$.<init>(build.sbt:1)
	at $92ac103c05862892d2d9$.<clinit>(build.sbt)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at sbt.compiler.Eval$.getModule(Eval.scala:608)
	at sbt.compiler.EvalDefinitions.values(Eval.scala:59)
	at sbt.internal.DefinedSbtValues.$anonfun$values$1(LoadedSbtFile.scala:51)
	at scala.collection.immutable.List.flatMap(List.scala:366)
	at sbt.internal.DefinedSbtValues.values(LoadedSbtFile.scala:51)
	at sbt.internal.EvaluateConfigurations$.$anonfun$evaluateSbtFile$4(EvaluateConfigurations.scala:167)
	at sbt.internal.Load$.loadSettingsFile$1(Load.scala:1120)
	at sbt.internal.Load$.$anonfun$discoverProjects$2(Load.scala:1129)
```

--- a/project/Build.scala
+++ b/project/Build.scala
@@ -142,7 +142,8 @@ object Build {
    *      - in stable release is always non-experimetnal
    */
   val expectedTastyVersion = "28.7-experimental-1"
-  checkReleasedTastyVersion()
+  // TODO: Restore test when upgrading reference compiler to 3.7.0 stable
+  // checkReleasedTastyVersion()
 
   /** Final version of Scala compiler, controlled by environment variables. */
   val dottyVersion = {

